src/components/material/MaterialPeriodical.tsx

```diff
@@ -32,12 +34,26 @@ const MaterialPeriodical: FC<MaterialPeriodicalProps> = ({
     })
     .flat();
 
-  return (
-    <MaterialPeriodicalSelect
-      groupList={groupObjectArrayByProperty(materialsPeriodical, "volumeYear")}
-      selectPeriodicalSelect={selectPeriodicalSelect}
-    />
+  const groupByvolumeYear = groupObjectArrayByProperty(
+    materialsPeriodical,
+    "volumeYear"
   );
+
+  const lastYear = Object.keys(groupByvolumeYear).sort().pop();
+  const lastEdition = lastYear ? groupByvolumeYear[lastYear].shift() : null;
+
+  if (lastYear && lastEdition) {
+    return (
+      <MaterialPeriodicalSelect
+        lastEdition={lastEdition as GroupListItem}
+        groupList={groupByvolumeYear as GroupList}
+        lastYear={lastYear}
+        selectPeriodicalSelect={selectPeriodicalSelect}
+      />
```

Comment:
Please move the `lastYear` and `lastEdition` inside the `MaterialPeriodicalSelect` component.

I do not understand why we need to define the last year and edition outside of the component and pass it as props. All the information is already available within the `groupList` prop.